packages/config/src/projects/layer2s/eclipse.ts

```diff
@@ -1,26 +1,36 @@
-import { upcomingL2 } from './templates/upcoming'
+import { EthereumAddress, UnixTime } from '@l2beat/shared-pure'
+import { underReviewL2 } from './templates/underReview'
 import { Layer2 } from './types'
 
-export const eclipse: Layer2 = upcomingL2({
+export const eclipse: Layer2 = underReviewL2({
   id: 'eclipse',
   display: {
     name: 'Eclipse',
     slug: 'eclipse',
     description:
       'Eclipse is a Layer 2 powered by the Solana Virtual Machine (SVM).',
```

Comment:
I'm still not sure if we stick to our L2 definition according to the FAQ, so leaving a comment just in case we do